apps/leo_gateway/test/leo_nfs_integration_tests.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
```

Comment:
Please add the explanation about the precondition to run this test like
- mkdir /ssd
- extract .tar.gz into /ssd/
- modify obj_container.path in leo_storage.conf